client/my-sites/comments/style.scss

```diff
@@ -79,6 +79,10 @@
 	overflow: hidden;
 	text-overflow: ellipsis;
 	white-space: nowrap;
+
+	span {
```

Comment:
If it's not too much trouble could we add classname to the span to avoid the specificity here? We usually like to avoid bare element selectors unless there's lots of them 😄